[2023-10-29T02:54:09.181+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2023-10-29T02:54:03.055953+00:00 [queued]>
[2023-10-29T02:54:09.188+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2023-10-29T02:54:03.055953+00:00 [queued]>
[2023-10-29T02:54:09.188+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-10-29T02:54:09.202+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): tsk_load_to_s3> on 2023-10-29 02:54:03.055953+00:00
[2023-10-29T02:54:09.205+0000] {standard_task_runner.py:57} INFO - Started process 1690 to run task
[2023-10-29T02:54:09.208+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'zillow_analytics_dag', 'tsk_load_to_s3', 'manual__2023-10-29T02:54:03.055953+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/analytics.py', '--cfg-path', '/tmp/tmpmc4svo07']
[2023-10-29T02:54:09.209+0000] {standard_task_runner.py:85} INFO - Job 36: Subtask tsk_load_to_s3
[2023-10-29T02:54:09.244+0000] {task_command.py:416} INFO - Running <TaskInstance: zillow_analytics_dag.tsk_load_to_s3 manual__2023-10-29T02:54:03.055953+00:00 [running]> on host ip-172-31-56-104.ec2.internal
[2023-10-29T02:54:09.302+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='chattisowmya@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='zillow_analytics_dag' AIRFLOW_CTX_TASK_ID='tsk_load_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2023-10-29T02:54:03.055953+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-29T02:54:03.055953+00:00'
[2023-10-29T02:54:09.303+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-29T02:54:09.303+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'aws s3 mv /home/ubuntu/response_data_29102023025406.json s3://zillowdatabucket/']
[2023-10-29T02:54:09.309+0000] {subprocess.py:86} INFO - Output:
[2023-10-29T02:54:09.921+0000] {subprocess.py:93} INFO - Completed 53.8 KiB/53.8 KiB (316.4 KiB/s) with 1 file(s) remainingmove: ../../home/ubuntu/response_data_29102023025406.json to s3://zillowdatabucket/response_data_29102023025406.json
[2023-10-29T02:54:09.979+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-10-29T02:54:09.998+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=zillow_analytics_dag, task_id=tsk_load_to_s3, execution_date=20231029T025403, start_date=20231029T025409, end_date=20231029T025409
[2023-10-29T02:54:10.022+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-29T02:54:10.043+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
